recipes/csaps/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "csaps" %}
+{% set version = "0.11.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 977644db0aa1a10b9e99c7159ea7f70b064310b8fc4bfd8fc6d293b0bfc3c385
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - numpy >=1.11.0
+    - pip
+    - python
+    - scipy >=1.0.0
+  run:
+    - {{ pin_compatible('numpy') }}
```

I did not check this old version but 1.1.0 doesn't seem to have any compiled extenstions.

```suggestion
    - {{ pin_compatible('numpy') }}
```

---

Did you mean to remove this line as a suggestion?

---

I used grayskull for that version and got:

```
    - python >=3.7,<4.0
    - numpy >=1.11.0,<1.20.0
    - scipy >=1.0.0,<1.6.0
```

Can you adjust that here?

PS: you can do specific versions with `grayskull pypi --strict-conda-forge "csaps=0.11.0"`.

---

I regenerated with grayskull and pushed.  Thanks for the pointer about the older version (and `grayskull`).